packages/fxa-auth-server/lib/senders/emails/templates/postVerify/index.ts

```diff
@@ -0,0 +1,24 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { appBadges, button, automatedEmailNoAction } from '../../partials';
+
+export const render = () => `
+  <mj-include path="./lib/senders/emails/css/global.css" type="css" css-inline="inline" />
+   <mj-section>
+     <mj-column>
+       <mj-text css-class="primary-text"><span data-l10n-id="postVerify-subdescription">Firefox Account verified. You're almost there.</span></mj-text>
+       <mj-text css-class="header-text"><span data-l10n-id="postVerify-title">Next sync between your devices!</span></mj-text>
+       <mj-image css-class="sync-logo" alt="Devices" src="https://accounts-static.cdn.mozilla.net/other/graphic-laptop-mobile.png"></mj-image>
```

Comment:
I think we need to cap the height of this image as it looks a little distorted, as well as add space below it -- left is original right is new template

<img width="1791" alt="Screen Shot 2021-08-17 at 3 08 17 PM" src="https://user-images.githubusercontent.com/6392049/129778190-8854b6fd-226d-46ba-b800-990b7da82782.png">

Reply:
Ohhh yes, thanks for bringing it to my notice!